### PR TITLE
Add long task to the event types

### PR DESCRIPTION
**Pull request by plousada**
### What and why?

Adds long tasks to the types of event list. 

### How?

A brief description of implementation details of this PR.

### Review checklist
- [ ] Feature or bugfix MUST have appropriate tests (unit, integration)
- [ ] Make sure each commit and the PR mention the Issue number or JIRA reference
- [ ] Add CHANGELOG entry for user facing changes

### Custom CI job configuration (optional)
- [ ] Run unit tests
- [ ] Run integration tests
- [ ] Run smoke tests

### CODE REVIEW

**Review by maxep — APPROVED** [review]

File: docs/rum_collection/data_collected.md [hunk 1]
```diff
@@ -24,7 +24,9 @@ There are additional [metrics and attributes that are specific to a given event
...
 | Error      | 30 days   | An error represents an exception or crash emitted by the mobile application attached to the view it is generated in.                                                                                                                                                                                        |
-| Action     | 30 days   | An action represents user activity in your mobile application (for example, application launch, tap, swipe, or back). Each action is attached with a unique `action.id` attached to the view it gets generated in.                                                                                                      |
+| Action     | 30 days   | An action represents user activity in your mobile application (for example, application launch, tap, swipe, or back). Each action is attached with a unique `action.id` attached to the view it gets generated in.                                                                                                                                              |
```
Comment (buraizu) [inline docs/rum_collection/data_collected.md:27]:
```suggestion
| Action     | 30 days   | An action represents user activity in your mobile application (for example, application launch, tap, swipe, or back). Each action is attached with a unique `action.id` to the view it gets generated in.                                                                                                                                              |
```

**Review by buraizu — CHANGES_REQUESTED** [review]
Thanks, the new information looks fine, but it looks like a word is mistakenly repeated in the adjacent section -- could you confirm and update if needed?

**Review by ncreated — APPROVED** [review]
👌

File: docs/rum_collection/data_collected.md (hunk 1 above)
Comment (plousada) [reply docs/rum_collection/data_collected.md:27]: I believe the current text makes the most sense although I admit it's not particular clear. 
It's trying to say that an `action.id` gets attached to an Action, and that an Action gets attached to a View.

Comment (buraizu) [reply docs/rum_collection/data_collected.md:27]: Thanks for clarifying, just made a suggestion which can hopefully clarify this for some customers as well. I recognize it's not part of your update, so I'll approve this request and let you decide if you want to apply the change.

**Pull request merged**